src/main/java/bio/terra/service/auth/iam/sam/SamIam.java

```diff
@@ -216,9 +217,12 @@ CreateResourceRequestV2 createDatasetResourceRequest(
 
     CreateResourceRequestV2 req = new CreateResourceRequestV2().resourceId(datasetId.toString());
 
+    List<String> adminEmailList =
+        StringUtils.isBlank(samConfig.adminsGroupEmail())
+            ? List.of()
+            : List.of(samConfig.adminsGroupEmail());
     req.putPoliciesItem(
-        IamRole.ADMIN.toString(),
-        createAccessPolicyOne(IamRole.ADMIN, samConfig.adminsGroupEmail()));
+        IamRole.ADMIN.toString(), createAccessPolicy(IamRole.ADMIN, adminEmailList));
```

Comment:
If the list is empty, is there any need to call `putPoliciesItem()`?

Reply:
it creates the policy in sam with an empty membership list, I did not want to fiddle with the policy structure

Reply:
FWIW @okotsopoulos had a similar PR that removed this completely https://github.com/DataBiosphere/jade-data-repo/pull/1647

Reply:
I think this is okay to leave as is for now.  That PR of mine was an exploratory draft: we'll need to do a bit more work to ultimately get tests to pass on BEEs without an admin group is what I found, so if we want to we can revisit it then.

Reply:
But, it would be nice to sidestep a possible gotcha and pull out this logic into a helper method.  I see it repeats throughout.

```
  private CreateResourceRequestV2 addAdminPolicy(CreateResourceRequestV2 req) {
    List<String> adminEmailList =
        StringUtils.isBlank(samConfig.adminsGroupEmail())
            ? List.of()
            : List.of(samConfig.adminsGroupEmail());
    return req.putPoliciesItem(
        IamRole.ADMIN.toString(), createAccessPolicy(IamRole.ADMIN, adminEmailList));
  }
```

Reply:
I deduped slightly differently, not for any good reason, I think I just stopped reading after "helper method" 😳